k8s_autoscale.go

```diff
@@ -17,7 +17,7 @@ import (
 const (
 	statsK8sIncreaseEventsCount = "increase_events"
 	statsK8sDecreaseEventsCount = "decrease_events"
-	statsK8sErrorsCount         = "errors"
+	statsK8sErrorsCount         = "k8s_errors"
```

Comment:
Yeah, should I do that for the other one as well? I particular the `batch` and `eval`